csharp/Adapter/Microsoft.Spark.CSharp/Network/ISocketWrapper.cs

```diff
@@ -38,6 +38,18 @@ internal interface ISocketWrapper : IDisposable
         Stream GetStream();
```

Comment:
do we still need this method given that you have added methods for getting input and output stream separately?